crates/pixi_build_frontend/src/protocols/builders/pixi.rs

```diff
@@ -80,12 +80,17 @@ impl ProtocolBuilder {
         workspace_manifest: WorkspaceManifest,
         package_manifest: PackageManifest,
     ) -> Self {
+        let configuration = package_manifest.build.configuration.clone().map(|v| {
+            v.deserialize_into()
+                .expect("Configuration dictionary should be serializable to JSON")
```

Comment:
This should be fine because we convert from `serde_value` to `serde_json` (to a dictionary with no fixed shape), and we come from TOML so all types should be good.